wavelength-plugins/profile.js

```diff
@@ -170,48 +176,47 @@ exports.commands = {
 	customtitle: {
 		set: 'give',
 		give: function (target, room, user) {
-			if (!this.can('declare')) return false;
+			if (!this.can('ban')) return false;
 			target = target.split(',');
 			if (!target || target.length < 3) return this.parse('/help', true);
 			let userid = toId(target[0]);
 			let targetUser = Users.getExact(userid);
 			let title = target[1].trim();
-			if (Db.customtitles.has(userid) && Db.titlecolors.has(userid)) {
-				return this.errorReply(userid + " already has a custom title.");
+			if (Db.titles.has(userid) && Db.titlecolors.has(userid)) {
+				return this.errorReply(`${userid} already has a custom title.`);
 			}
 			let color = target[2].trim();
-			if (color.charAt(0) !== '#') return this.errorReply("The color needs to be a hex starting with '#'.");
-			Db.customtitles.set(userid, [title, color]);
+			if (color.charAt(0) !== '#') return this.errorReply(`The color needs to be a hex starting with "#".`);
+			Db.titles.set(userid, [title, color]);
 			if (Users.get(targetUser)) {
 				Users(targetUser).popup(
-					'|html|You have received a custom title from ' + WL.nameColor(user.name, true) + '.' +
-					'<br />Title: ' + showTitle(toId(targetUser)) +
-					'<br />Title Hex Color: ' + color
+					`|html|You have received a custom title from ${WL.nameColor(user.name, true)}.` +
+					`<br />Title: ${showTitle(toId(targetUser))}` +
+					`<br />Title Hex Color: ${color}`
 				);
 			}
-			this.logModCommand(user.name + " set a custom title to " + userid + "'s profile.");
-			Monitor.adminlog(user.name + " set a custom title to " + userid + "'s profile.");
-			return this.sendReply("Title '" + title + "' and color '" + color + "' for " + userid + "'s custom title have been set.");
+			this.addModAction(`${user.name} set a custom title to ${target[0]}'s profile.`);
```

Comment:
Why are we displaying the setting of the profile publicly in the room it was used in? Its already being logged as needed.